### ISSUE TITLE
Execution error 

### ISSUE THREAD

**Issue by tim-hilde** [alfredapp/banner-be-gone-workflow#1]
### Workflow version

2023.1

### Alfred version

5.1.3

### macOS version

14.0

### Debugger output

```alfred_debugger
/Users/tim/Library/Caches/com.runningwithcrayons.Alfred/Workflow Scripts/004B9822-D801-47CF-B467-00385C51B671: execution error: Error: TypeError: undefined is not an object (evaluating 'banner.perform') (-2700)
```

### More details

I tried using the workflow

**Comment by vitorgalvao** [comment]
What language is your system?

**Comment by tim-hilde** [comment]
Its German. I already changed the settings to match the German translation

**Comment by vitorgalvao** [comment]
If you translated them directly, they may be incorrect. It depends on what macOS uses. Which words did you put?

**Comment by tim-hilde** [comment]
I used the translation macOS uses: 
<img width="702" alt="image" src="https://github.com/alfredapp/banner-be-gone-workflow/assets/44113468/892b53d7-24b3-4466-a208-d2c29f9789fd">

from https://support.apple.com/de-de/guide/mac-help/mchl2fb1258f/mac

**Comment by vitorgalvao** [comment]
Good thinking. Unfortunately those are incorrect (in the English version too). Do `Schließen` and `Alle entfernen`.

**Comment by tim-hilde** [comment]
Haha, classic. Unfortunately, these keywords still lead to an error

**Comment by vitorgalvao** [comment]
I can’t reproduce the error, I changed my system to German and those work fine. You do have notifications showing, don’t you? Otherwise the error is normal.

**Comment by tim-hilde** [comment]
I just did the same and changed my system to English and it still doesn't work. On the other hand the error has changed:

[11:48:27.239] ERROR: Banner Be Gone[Run Script] /Users/tim/Library/Caches/com.runningwithcrayons.Alfred/Workflow Scripts/E3AB1A3A-8B4E-4183-93D7-BE8824A46CDB: execution error: Error: Error: Invalid index. (-1719)

**Comment by vitorgalvao** [comment]
I’ll need a video of your screen, with the debugger open, to diagnose further. You can [record your screen with built-in macOS tools](https://support.apple.com/en-us/HT208721).

**Comment by tim-hilde** [comment]
Sure

https://github.com/alfredapp/banner-be-gone-workflow/assets/44113468/124cac35-0a6c-4986-adae-2bd252ca0a63

**Comment by vitorgalvao** [comment]
Your notifications aren’t visible. The workflow doesn’t clear notifications from the Notification Centre (those are auto-cleared every week), it hides the notifications currently visible.

**Comment by vitorgalvao** [comment]
Just released 2023.2 which mentions in the About that notifications have to be visible. Also it speeds up execution and removes the language option (should now work regardless of language).

To note that the workflow *can* remove those notifications from the Notification Centre, but they need to be visible. For that you’ll need to use the Hotkey instead of the Keyword, since showing Alfred hides the Notification Centre.
